test_conformance/math_brute_force/CMakeLists.txt

```
@@ -43,6 +43,8 @@ set(${MODULE_NAME}_SOURCES
 
 # math_brute_force compiles cleanly with -Wall (except for a few remaining
 # warnings), but other tests not (yet); so enable -Wall locally.
-set_gnulike_module_compile_flags("-Wall -Wno-strict-aliasing -Wno-unknown-pragmas")
+set_gnulike_module_compile_flags("-Wall -Wno-strict-aliasing -Wno-unknown-pragmas -ffp-contract=off")
+
+add_cxx_flag_if_supported("-ffp-contract=off")
```

Minor, but if you're fixing the issue above, I'd consider fixing this too: the other uses of add_cxx_flag_if_supported do not use quotes:

```suggestion
add_cxx_flag_if_supported(-ffp-contract=off)
```